### ISSUE TITLE
1660 buildings with sidewalks/curbs

### ISSUE THREAD

**Issue by tedbarnett** [TimeWalkOrg/Manhattan_public#34]
The Castello Plan artwork depicts narrow sideways and sometimes low walls around the blocks of homes in 1660 New Amsterdam.  Should we include these in the terrain or with the buildings?  I would argue group this info with the buildings so that terrain can just be the natural ground (i.e. not including "building foundations").

We need to build narrow sidewalks and/or curbs next to the buildings in 1660 Manhattan.

![image](https://github.com/TimeWalkOrg/Manhattan/assets/694013/b3959c9e-7513-45ee-9e9a-5ed650a86981)

**Comment by JustPineStudio** [comment]
We would suggest to instance all of that on a spline. That way we can go around and add them where we like, as well as use the same system on the other timelines with just changing the 3d models that are instanced. Which would make us have more control and easy ajustment where needed.

**Comment by tedbarnett** [comment]
Agreed. Let's leave that to Dario who has been building spline-controlled fences and other objects.
